### ISSUE TITLE
Changing names on checklists breaks previously used checklist

### ISSUE THREAD

**Issue by mjaakko** [UniversityOfHelsinkiCS/labtool#722]
This is probably because checklists are saved to the database as JSON;

```javascript
const Checklist = sequelize.define(
    'Checklist',
    {
      week: DataTypes.INTEGER,
      list: DataTypes.JSONB,
      courseName: DataTypes.STRING,
      master: DataTypes.BOOLEAN
    },
...
```

**Comment by hisahi** [comment]
Shouldn't this be in the project? Is it on sprint 4?

**Comment by hisahi** [comment]
Fixed in #804
